icons/treehouse.svg

```diff
@@ -1 +1 @@
-<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Treehouse icon</title><path d="M20.537 4.118c-.806-.453-2.092.278-2.871 1.635L16.25 8.215a3.104 3.104 0 0 0 .21 3.18l.041.062c.653.94 1.535 1.808 1.823 2.118a1.613 1.613 0 0 1-.739 2.654 1.603 1.603 0 0 1-2.025-1.747c.045-.35-.067-.927-.574-1.489-.506-.563-1.54.5-1.874 1.61l-.016.061c-.334 1.094-.546 2.05-.482 2.143.037.06.072.12.105.182a1.81 1.81 0 0 1-3.196 1.701 1.806 1.806 0 0 1 .747-2.446l.121-.061c.065-.03.26-.486.423-1.032l.301-.987c.019-.047.033-.107.045-.168l.897-3.19-.957 1.96c-.112-.363-.3-.38-.709-.091-.243.183-.653.531-.85.669-.365.273-.685.788-.851 1.109a1.313 1.313 0 0 1-.41.5c-.684.564-1.687.456-2.234-.227a1.591 1.591 0 0 1 .912-2.552c.409-.092 1.777-.927 2.596-1.52.152-.106.274-.197.38-.304l2.203-1.67-1.914 1.032s-.196-.016-.426.017c-.698.075-1.428.182-1.564.35a.999.999 0 0 1-.29.272c-.637.456-1.519.32-1.989-.317A1.437 1.437 0 0 1 6.29 8.04c.259-.183.577-.274.865-.274.518.016 1.87.29 2.993.092l.288-.045c1.14-.196 2.476-1.186 3.024-2.187l1.184-2.067c.653-1.139.608-2.384-.105-2.795l-1.323-.76c-.653-.363-1.715-.363-2.354 0L2.004 4.97C1.337 5.319.805 6.23.805 6.975v9.744c0 .744.532 1.656 1.178 2.02l8.85 4.983c.652.365 1.716.365 2.354 0l8.826-4.983c.653-.368 1.184-1.276 1.184-2.02v-9.76c0-.744-.531-1.653-1.169-2.02l-1.46-.823"/></svg>
\ No newline at end of file
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Treehouse icon</title><path d="M20.443 4.339c-.797-.449-2.07.275-2.84 1.617l-1.4 2.435a3.07 3.07 0 0 0 .207 3.145l.041.062c.646.93 1.518 1.788 1.803 2.095a1.595 1.595 0 0 1-.73 2.625 1.586 1.586 0 0 1-2.004-1.728c.045-.346-.066-.917-.567-1.473-.501-.557-1.524.495-1.854 1.592l-.016.06c-.33 1.083-.54 2.028-.477 2.12.037.06.072.12.104.18a1.79 1.79 0 0 1-3.161 1.683 1.786 1.786 0 0 1 .739-2.42l.12-.06c.064-.03.257-.48.418-1.02l.298-.977a.96.96 0 0 0 .044-.166l.887-3.155-.946 1.938c-.111-.359-.297-.375-.701-.09-.24.181-.646.526-.841.662-.361.27-.678.78-.842 1.097a1.299 1.299 0 0 1-.406.495 1.558 1.558 0 0 1-2.21-.225 1.574 1.574 0 0 1 .903-2.524c.404-.091 1.758-.917 2.568-1.504.15-.104.27-.194.375-.3l2.18-1.652-1.894 1.02s-.194-.015-.421.017c-.69.075-1.412.18-1.547.347a.988.988 0 0 1-.287.269 1.415 1.415 0 0 1-1.967-.314 1.421 1.421 0 0 1 .332-1.972c.256-.181.57-.271.856-.271.512.016 1.85.287 2.96.09l.285-.044c1.128-.194 2.45-1.173 2.991-2.163l1.171-2.044c.646-1.127.602-2.358-.104-2.765L13.202.269c-.646-.359-1.697-.359-2.329 0L2.112 5.181C1.452 5.526.926 6.428.926 7.164v9.638c0 .736.526 1.638 1.165 1.998l8.754 4.93c.645.36 1.697.36 2.328 0l8.73-4.93c.646-.364 1.171-1.262 1.171-1.998V7.15c0-.736-.525-1.635-1.156-1.998l-1.444-.814"/></svg>
```

Comment:
It looks like they've now smoothed out the logo a little bit - might need to adjust the SVG...
```svg
M20.398,4.386c-0.821-0.448-2.09,0.299-2.837,1.642l-1.418,2.389c-0.523,0.97-0.448,2.165,0.224,3.135l0.075,0.075 	c0.672,0.896,1.493,1.792,1.792,2.09c0.149,0.149,0.299,0.373,0.373,0.672c0.224,0.821-0.224,1.717-1.12,1.941 	c-0.821,0.224-1.717-0.224-1.941-1.12c-0.075-0.224-0.075-0.448-0.075-0.597c0.075-0.373-0.075-0.896-0.597-1.493 	c-0.523-0.523-1.493,0.523-1.866,1.568v0.075c-0.299,1.045-0.523,2.016-0.448,2.09c0,0.075,0.075,0.075,0.075,0.149 	c0.448,0.896,0.149,1.941-0.747,2.389s-1.941,0.149-2.389-0.747c-0.448-0.896-0.149-1.941,0.747-2.389 	c0.075,0,0.075-0.075,0.149-0.075c0.075,0,0.224-0.448,0.448-1.045c0.149-0.523,0.224-0.747,0.299-0.97 	c0.075-0.299,0.224-1.045,0.149-1.344c-0.075-0.373-0.448-0.373-0.821-0.149c-0.224,0.149-0.672,0.523-0.821,0.672 	c-0.373,0.299-0.672,0.747-0.821,1.12c-0.075,0.149-0.224,0.373-0.373,0.523c-0.672,0.523-1.642,0.448-2.24-0.224 	c-0.523-0.672-0.448-1.642,0.224-2.24c0.224-0.149,0.448-0.299,0.672-0.299c0.373-0.075,1.717-0.896,2.538-1.493 	c0.149-0.075,0.373-0.299,0.448-0.373c0.224-0.299,0-0.523-0.224-0.523C9.201,9.91,8.454,9.984,8.305,10.208 	c-0.075,0.075-0.149,0.224-0.299,0.299c-0.597,0.448-1.493,0.299-1.941-0.299C5.617,9.611,5.767,8.715,6.364,8.267 	c0.224-0.224,0.597-0.299,0.821-0.299c0.523,0,1.866,0.299,2.911,0.075l0.299-0.075c1.12-0.224,2.463-1.194,2.986-2.165 	c0,0,0.523-0.896,1.12-2.016c0.672-1.12,0.597-2.389-0.075-2.762L13.157,0.28c-0.672-0.373-1.642-0.373-2.314,0L2.184,5.207 	C1.512,5.58,0.989,6.476,0.989,7.222v9.555c0,0.747,0.523,1.642,1.194,2.016l8.659,4.927c0.672,0.373,1.717,0.373,2.314,0 	l8.659-4.927c0.672-0.373,1.194-1.269,1.194-2.016V7.222c0-0.747-0.523-1.642-1.194-2.016C21.816,5.207,21.219,4.834,20.398,4.386z
```

![Uploading image.png…]()

Reply:
@adamrusted Done. A precision of 3 didnt't make it through the linter, so I had to increase it to 4.